### PR TITLE
Update domains.txt

### PR DESCRIPTION
**Pull request by ovflowd**
Added some domains from Brazil Government (Federal, State and Institutes)

### CODE REVIEW

File: config/domains.txt [hunk 1]
```diff
@@ -20574,3 +20574,93 @@ uintacounty.com
 wrightwyoming.com
 wyomingbusiness.org
 wyomingtourism.org
+
+// Brazil Federal Government
+eb.mil.br
```
Comment (benbalter) [inline config/domains.txt]: `.gov.br` and `.mil.br` are already listed and would include any subdomains. I believe if you add `def.br` and `mp.br`, this entire list should be included.

File: config/domains.txt [hunk 2]
```diff
@@ -20574,3 +20574,93 @@ uintacounty.com
...
+senado.gov.br
+camara.gov.br
+
+// Brazil Federal Institutes
+anatel.gov.br
+regulacao.gov.br
+anac.gov.br
+abin.gov.br
+unb.br
+ufac.br
```
Comment (benbalter) [inline config/domains.txt]: For all `u*.br`, these looks like universities. Even if federally funded, would it be accurate to say any user with e.g., a `ufac.br` email address is a government employee?

**Comment by ovflowd** [comment]
Hey @benbalter I'm going to remove those to duplicate. In Brazil the federal University Employee's are actually Public Federal employee's.

In Brazil works in this way, the federal universities are part of the federal government.

File: config/domains.txt (hunk 2 above)
Comment (ovflowd) [reply config/domains.txt]: Hey @benbalter I'm going to remove those to duplicate. In Brazil the federal University Employee's are actually Public Federal employee's.

In Brazil works in this way, the federal universities are part of the federal government.

Comment (ovflowd) [reply config/domains.txt]: Actually, the onliest subdomain from those domains that may be used as non employee, could be the students subdomain, like `aluno.unb.br`, so I don't know how do a reverse rule.

File: config/domains.txt (hunk 1 above)
Comment (ovflowd) [reply config/domains.txt]: So I just need to add, `gov.br` and it will actually add all subdomains that terminate with `gov.br.`?

**Comment by jbjonesjr** [comment]
@benbalter what do you think at this point?

**Comment by benbalter** [comment]
For now I think it'd be best to remove the university addresses, since they may contain non-employee students, so that we can add the other domains.

**Comment by ovflowd** [comment]
@benbalter if you want I can remove those, but generally the University Students have a specific combination of Addresses, like `@ aluno . name-of-university . br`

And all the rest are for employees, staffs, etc. Remembering that all employees from federal universities are employees from the federal government.

That's something that I only see here in Brazil and it's very strange.

**Pull request closed without merging**